samples/CosmosDbIntegtation/ReadThroughSamples/ListSample.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.Functions.Worker.Extensions.Redis;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using StackExchange.Redis;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Policy;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public static class ListSample
+    {
+        //Redis Cache primary connection string from local.settings.json
+        public const string redisConnectionString = "redisConnectionString";
+        private static readonly IDatabase s_redisDb = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(redisConnectionString)).GetDatabase();
```

Comment:
You can attempt to do a Lazy load of this if you're interested in learning about it [learn.microsoft.com/en-us/dotnet/framework/performance/lazy-initialization](https://learn.microsoft.com/en-us/dotnet/framework/performance/lazy-initialization)